docs/customizations/parts/operation-tags-slot-example.md

```
@@ -0,0 +1,17 @@
+
+useTheme({
+    operation: {
+      slots: [
+        ...DEFAULT_OPERATION_SLOTS,
+        'tags',
+      ],
+    },
+})
+</script>
+
+<OASpec :isDark="isDark" />
```

Markdownlint (0.35.0)

Element: OASpec
Inline HTML

(MD033, no-inline-html)